.gitignore

```diff
@@ -68,6 +68,9 @@ dist/
 # Ignore build www directories
 www/
 
+# Ignore build documentation directories
+_book/
```

Comment:
this is the only ignored folder that starts with an underscore, should it just be book/?

Reply:
Love it if this wasn't the case.  but, couldn't change that.  In either case this is obsolete now that we're using docusaurus.